portability-spi-transfer/src/main/java/org/datatransferproject/spi/transfer/provider/converter/PhotoToMediaConversionExporter.java

```diff
@@ -0,0 +1,37 @@
+package org.datatransferproject.spi.transfer.provider.converter;
+
+import org.datatransferproject.spi.transfer.provider.Exporter;
+import org.datatransferproject.spi.transfer.provider.ExportResult;
+import org.datatransferproject.types.common.ExportInformation;
+import org.datatransferproject.types.common.models.media.MediaContainerResource;
+import org.datatransferproject.types.common.models.photos.PhotosContainerResource;
+import org.datatransferproject.types.transfer.auth.AuthData;
+import org.datatransferproject.types.transfer.auth.TokensAndUrlAuthData;
+
+import java.util.Optional;
+import java.util.UUID;
+
+public class PhotoToMediaConversionExporter<
+      A extends AuthData,
+      PCR extends PhotosContainerResource,
+      PE extends Exporter<A, PCR>
+    > implements Exporter<A, MediaContainerResource> {
+  private final PE wrappedPhotoExporter;
+  public PhotoToMediaConversionExporter(PE wrappedPhotoExporter) {
+    this.wrappedPhotoExporter = wrappedPhotoExporter;
+  }
+
+  public ExportResult<MediaContainerResource> export(
+      UUID jobId,
+      A authData,
+      Optional<ExportInformation> exportInformation
+  ) throws Exception {
+    ExportResult<PCR> originalExportResult =
+        wrappedPhotoExporter.export(jobId, authData, exportInformation);
+    PCR photosContainer = originalExportResult.getExportedData();
+    MediaContainerResource mediaContainerResource =
+        MediaContainerResource.photoToMedia(photosContainer);
+    return new ExportResult<>(originalExportResult.getType(), mediaContainerResource,
```

Comment:
Could make this a convenience method on the ExportResult itself.
```suggestion
    return originalExportResult.copyWithContainerResource(mediaContainerResource);
```

Reply:
Rrr.. wait I seem to have messed up a commit somewhere. Let me see where this change went (I definitely already did this).

Reply:
ah no i was viewing our conversation against an outdated diff. this **is** indeed completed.